tests/drivers/uart/uart_basic_api/testcase.yaml

```diff
@@ -1,13 +1,9 @@
 tests:
 -   test_uart:
-        arch_whitelist: x86
-        platform_whitelist: quark_se_c1000_devboard
-            arduino_101 quark_d2000_crb
+        platform_whitelist: quark_se_c1000_devboard quark_d2000_crb
         tags: drivers
 -   test_uart_shell:
-        arch_whitelist: x86
         build_only: true
         extra_args: CONF_FILE=prj_shell.conf
-        platform_whitelist: quark_se_c1000_devboard
-            arduino_101 quark_d2000_crb
+        platform_whitelist: quark_se_c1000_devboard quark_d2000_crb
```

Comment:
the arduino 101 is covered by quark_se_c1000_devboard for this particular driver test, the QA folks were using the whitelist for their own accounting. Ultimate goal is to remove this type of whitelisting and and provide more coverage by matching features between board and what the testscase covers.